draft-ietf-quic-recovery.md

```diff
@@ -398,8 +417,80 @@ A packet sent on an RTO alarm MUST NOT be blocked by the sender's congestion
 controller. A sender MUST however count these bytes as additional bytes in
 flight, since this packet adds network load without establishing packet loss.
 
+## Multiple Packet Number Space Optimizations {#optimizations}
+
+There are cases where one may be able to gain recovery information from
+acknowledgements of packets in another packet number space, but they rely
+on complex assumptions about the peer’s processing and acknowledgement
+algorithms.  Even those are unable to quickly recover from cases such as
+losing the client's Initial, but receiving the 0-RTT packets.  Below are
+three different optimizations in increasing complexity that minimize
+handshake latency.
+
+### Empty ACK frames
```

Comment:
Copying my comment I made on #1:

What is an empty ACK? Every ACK encodes a Largest Acknowledged, and since for some reason we decided that QUIC packet numbers start at 0 (I still thing this was a terrible decision. It added a whole range of special cases to my loss recovery code.), we can't use that as a sentinel value as well.